src/i18n/en/index.ts

```diff
@@ -4,7 +4,7 @@ const en: BaseTranslation = {
 	language: 'English',
 	happyHolidays: 'Happy Holidays!',
 	christmasIsComing: '{ date|fullDay } is in { time }',
-	day: '{{ one day | ?? days}}' // not sure if there's a way to name this parameter
+	day: '{{ days: one day | ?? days }}'
```

Comment:
yes you can ;)

Reply:
But I saw that there is a bug in the type generator.:
`day` in `RootTranslation` should be typed as `RequiredParams<'days'>` and not just `string`. It seems I didn't account for the case that just a `PluralPart` gets defined.
I'll open an issue in `typesafe-i18n`.